change/@internal-react-composites-ec7e402b-767e-4419-b47a-0546059b1958.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "optimization",
```

Comment:
I'd recommend adding a more informative comment. Especially because when we next release, we'll want to know "oh we already cherry-picked that one into the last release.